docs/react_routes.md

```diff
@@ -0,0 +1,14 @@
+# React Routes
+| Route| Require Auth| Description|
+| -| -| -
+| `/`| No| view LandingPage
+| `/login`| No| view Login
+| `/signup`| No| view Signup
+| `/home`| No| view Home
```

Comment:
I think /home route should require Auth

Reply:
@AymanAlqoqa   why?

Reply:
@AymanAlqoqa  I suggest leaving the ```auth``` to the end. :)